### DISCUSSION TITLE
リスト内に特定の書式でコードを書くと若干画面レイアウトが崩れる

### DISCUSSION THREAD

**Discussion by ichii731** [increments/qiita-discussions#109] [要望・アイデア・不具合]
本当に些細なことをある記事を閲覧中に見つけましたので一応ですがDiscussionにて連絡させていただきます。
`- `で入力するリストの中に4スペースのインデントを手動で無理やり挿入して、コードを記述すると該当コードの左側に余白が出来てしまい、モバイルでの表示が若干おかしくなっているようです。
書き方に癖がありすぎるので、スマホで以下限定共有記事で観ていただければわかると思います。

https://qiita.com/ichii731/private/7774de1966aa927e0cc7

本当に変な書き方をしないといけないので、許容範囲内の例外として既に認識なさっているのでしたら申し訳ございませんm(_ _)m
ちなみにGoogleのモバイルフレンドリーテストでも特段問題は起きていません。
![image](https://user-images.githubusercontent.com/59349039/148738678-68fb04a3-0228-4c94-8ecb-3a866720bb9b.png)

**Comment by kskwtnk** [comment]
@ichii731 
ご意見ありがとうございます！

こちら、スタイルがおかしくなっているのを確認いたしました
Issueを作成して対応いたします！

**Comment by kskwtnk** [comment]
@ichii731 
本件修正してリリース完了しました！
ご準備いただいていた限定公開の記事も正しく表示できております！

**Reply by ichii731** [reply]
有難うございます！！
